includes/Rest.php

```diff
@@ -42,10 +42,5 @@ public static function verify_request_origin() {
 		if ( $request_url['host'] !== $site_url['host'] ) {
 			throw new Exception( 'Request host was not valid.' );
 		}
-
-		// Verify path.
```

Comment:
I think this check is valuable, because a host could have multiple WP sites. For example, "test.com/wp1" and "test.com/wp2"

Reply:
We should probably find a different way to check that then, because this check would always fail whenever requesting the article content via the API, which is the only way to asynchronously implement the server-side paywall (which means we don't get dinged with the increase page load times)